perun-cli/getAttribute

```diff
@@ -116,7 +118,45 @@ if (defined $entitylessSubj) {
 	defined ($uesId) and $parameters{userExtSource} = $uesId;
 }
 
-my @attributes = $attributesAgent->getAttribute( %parameters );
+# find if exists combination of entered parameters
+my $is=0;
+if (exists($parameters{'resource'}) and exists($parameters{'member'})) {
+	my $resourcesAgent = $agent->getResourcesAgent;
+	my @members = $resourcesAgent->getAllowedMembers( resource => $resourceId );		             	
+	foreach my $member ( @members) {
+		if ($member->getId == $memberId) { $is=1;}
+	}
+	if ($is == 0) { die "ERROR: member is not assigned to resource \n";}
+}
+
+if (exists($parameters{'resource'}) and exists($parameters{'group'})) {
+	my $resourcesAgent = $agent->getResourcesAgent;
+	my @groups = $resourcesAgent->getAssignedGroups( resource => $resourceId );		             	
+	foreach my $group ( @groups) {
+		if ($group->getId == $groupId) { $is=1;}
+	}
+	if ($is == 0) { die "ERROR: group is not assigned to resource \n";}
+}
+
+if (exists($parameters{'member'}) and exists($parameters{'group'})) {
+	my $groupsAgent = $agent->getGroupsAgent;
+	my @members = $groupsAgent->getGroupMembers( group => $groupId);;		             	
```

Comment:
I would suggest to get member by its ID and then it's group. Since member usually won't be inside thousands of Groups, but Group can have thousands of Members. Also we have ``isGroupMember(group,member)`` in ``GroupsManagerBlImpl``, So if you would implement Entry and RPC layer for this method, you could use it in a CLI and it would be 1 single select with boolean result.